vassal-app/src/main/java/VASSAL/counters/UsePrototype.java

```diff
@@ -323,6 +336,86 @@ public String getState() {
     public String getType() {
       return ID + nameConfig.getValueString();
     }
+
+    private String getValueString() {
+      return nameConfig.getValueString();
+    }
+
+    private class Validator extends JLabel {
```

Comment:
Should this be `private static`?

Reply:
Re-organized and re-coded slightly. Ready to go.